src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+            {
+                _MaxScale = DataRaw.Sum(x => Math.Abs(x));
+                _MinScale = 0;
+            }
+            else
+            {
+                _MaxScale = DataRaw.Max(x => Math.Abs(x));
+                var min = DataRaw.Min();
+                _MinScale = min < 0 ? min : 0;
+            }
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <param name="deep"></param>
+        public void ValidateData(bool deep = false)
+        {
+            if (GaugeLabels != null && GaugeLabels.Length != DataRaw.Length)
+                throw new InvalidOperationException("Gauge labels must match size of data values");
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public LegendItem[] GetLegendItems()
+        {
+            if (GaugeLabels is null)
+                return null;
+
+            List<LegendItem> legendItems = new List<LegendItem>();
+            for (int i = 0; i < GaugeLabels.Length; i++)
+            {
+                var item = new LegendItem()
+                {
+                    label = GaugeLabels[i],
+                    color = GaugeColors[i],
+                    lineWidth = 10,
+                    markerShape = MarkerShape.none
+                };
+                legendItems.Add(item);
+            }
+
+            return legendItems.ToArray();
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public AxisLimits GetAxisLimits() =>
+            (GaugeLabels != null) ? new AxisLimits(-3.5, 3.5, -3.5, 3.5) : new AxisLimits(-2.5, 2.5, -2.5, 2.5);
+
+        /// <summary>
+        /// Reduces an angle into the range [0°-360°]
+        /// </summary>
+        /// <param name="angle">Angle value</param>
+        /// <returns>Return the angle whithin [0°-360°]</returns>
+        private double ReduceAngle(double angle)
+        {
+            double reduced = angle;
+
+            if (angle > 360.0)
+                reduced -= 360 * (int)(angle / 360);
+            else if (angle < -360.0)
+                reduced += 360 * (int)(angle / 360);
+
+            return reduced;
+        }
+
+        /// <summary>
+        /// This is where the drawing of the plot is done
+        /// </summary>
+        /// <param name="dims">Plot dimensions</param>
+        /// <param name="bmp">Bitmap where the drawing is done</param>
+        /// <param name="lowQuality">Image quality</param>
+        public virtual void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
+        {
+            int numGroups = DataRaw.Length;
+            double minScale = new double[] { dims.GetPixelX(1), dims.GetPixelY(1) }.Min();  // Not sure why, but GetPixelX(1) returns a reasonable dimension to draw the plot
```

Comment:
I think you should do this:

```cs
double minScale = new double[] { dims.PxPerUnitX, dims.PxPerUnitX }.Min();
```

Reply:
Sure!
By the way, I seems that using `PxPerUnit` results in the plot being too small regarding the size of the whole control. Visually there are some chucks of empty space at both the plot's top and bottom. The same thing happens with `RadarPlot`.
That's the only reason I tried `GetPixel` instead: the plot seemed to better fill the control area.

Reply:
Actually I went with this 😎 
```cs
double minScale = Math.Min(dims.PxPerUnitX, dims.PxPerUnitY);
```